pkg/import/import_resources.go

```diff
@@ -90,7 +93,9 @@ func Import(rd, definitionsFile io.Reader) ([]string, []string, error) {
 			id = append(id, fmt.Sprintf("%s", after[field]))
 		}
 		arg := fmt.Sprintf("%s %s", resAddr, strings.Join(id, resourceParams.Separator))
+
 		if resourceParams.Priority == 1 {
```

Comment:
Is it possible for priority to exist and be different from 1 ?
In this case, the code would silently generate an incorrect script.